docs/batch_request.md

```diff
@@ -0,0 +1,30 @@
+## BatchRequest
+
+BatchRequest is useful when you want to make multiple requests efficiently. It batches all requests (upto 20 requests) into a json object and makes one api call. You can learn more about it on [Microsoft Docs](https://docs.microsoft.com/en-us/graph/json-batching). 
+
+## Code Sample
+
+```go
+import "github.com/microsoftgraph/msgraph-sdk-go-core"
+import abstractions "github.com/microsoft/kiota-abstractions-go"
+
+reqInfo := client.Me().CreateGetRequestInformation()
+batch := msgraphsdkcore.NewBatchRequest()
+batchItem := batch.AppendBatchItem(*reqInfo)
+
+resp, err := msgraphsdkcore.SendBatch(*batch, reqAdapter)
```

Comment:
I believe the experience could be improved here.

We should at least offer
```suggestion
resp, err := batch.Send(reqAdapter)
```
(with maybe a context parameter once the context feature is implemented)


We could also offer

```suggestion
resp, err := graphClient.SendBatch(batch)
```

I know the specification was not super explicit, and sorry about that. but what do you think?

Reply:
Sorry if I wasn't clear. I think the Send method should be on a receiver for batch instead of accepting the batch as a parameter.

Reply:
(on top of having another on available on the client)